src/elm/Crumbs.elm

```diff
@@ -241,7 +242,7 @@ toPath page =
                             ( org, Just <| Pages.OrgSecrets engine org Nothing Nothing )
 
                         nameCrumb =
-                            ( name, Nothing )
+                            ( Maybe.withDefault name <| percentDecode name, Nothing )
```

Comment:
the need to decode a team or name is to go alongside #171 that will encode urls for names that include special characters